node_modules/credscan-pkg/main.js

```diff
@@ -0,0 +1,57 @@
+const path = require('path');
+const config = require('./config.json');
+const os = require('os');
+var version = 'core'
+var namespace = 'CredScanSrc'
+if (version === 'core') version = 'coreapp';
+
+const baseNetAppPath = path.join(__dirname, '/src/' + namespace + '/bin/Debug/net' + version + '2.0');
+
+process.env.EDGE_USE_CORECLR = 1;
+if (version !== 'standard')
+    process.env.EDGE_APP_ROOT = baseNetAppPath;
+
+var edge = require('edge-js');
+var core = require('@actions/core');
```

Comment:
put all the require at top only

Reply:
Moved all require statements to the top

Reply:
The code stopped working when I did this